### ISSUE TITLE
Passwords are not decrypted in UI after enter passphrase, when open options before entering security passphrase

### ISSUE THREAD

**Issue by pinpontitit** [floccusaddon/floccus#1059]
### Which version of floccus are you using?

Floccus extension for firefox

### Sync method

{"label"=>"Google Drive"}

### Which browser are you using?

Librewolf

### Describe the Bug

Today when I checked the flocxus extension settings, I discovered that my passphrase had changed.
It became a sequence of like 30 random caracters like "H6Klm9y3OU0lhZ6jKL....."

I dont know if in consequence the syncing stopped or no, I just saw that my passphrase was replaced by that.

### Expected Behavior

I should have found my original passphrase the one I choosed.

### To Reproduce

I dont know if yiu would be able to rzproduce it.
I just went to check the settings and discivered that. The extension was installed and set up 2 days ago.

Is it a bug or are you trying to implement some encryption to the passphrase ?
Or is it revealing some encryption process that should be hidden ?

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
It could be some bug associated with credential protection (Unlock passphrase). Did you set that up as well?

**Comment by pinpontitit** [comment]
Hi
No, credential protection wasn't set up, and still isnt set up.
Today I went to check and my passphrase is back to normal.

**Comment by marcelklehr** [comment]
Then let's wait for it to reappear to pin point what's happening.

**Comment by pinpontitit** [comment]
Hi
So it happened again.
You're right it must be related with the Credential Lock function.

When I opened my browser, a tab with Floccus settings was open (maybe from previous session). On top of the settings was the pop-up asking for my Credentials and I could see behind it that my passphrase was looking pretty long (which I like because like this no one can see the length of my true passphrase).

After entering my credentials, the popup disappeared, and I could see that my passphrase was still very long. And by clicking in the Eye icon, to reveal it, I discovered that the passphrase was a series of random caracters.

After closing the tab, and opening the Floccus settings again, my passphrase was back to normal.

So it means that after unlocking my credentials, Floccus didnt changed the encrypted passphrase to my normal passphrase automatically. I had to close and open again Floccus settings.

Its not a big issue, maybe not an issue at all, but I wanted you to know.
